pallets/lbp/src/lib.rs

```diff
@@ -658,7 +663,13 @@ pub mod pallet {
 
 			Self::deposit_event(Event::LiquidityRemoved(pool_id, asset_a, asset_b, amount_a, amount_b));
 
-			Ok(())
+			 let amount_a_perbill: Balance = transfer_to_xyk * amount_a;
+			 let amount_b_perbill: Balance = transfer_to_xyk * amount_b;
+			 if amount_a_perbill.is_zero() || amount_b_perbill.is_zero() {
+				  Ok(())
+			 } else {
+				 T::OnRemoveLiquidity::on_end_of_lbp(who, asset_a, asset_b, amount_a_perbill, amount_b_perbill)
```

Comment:
what will happen if there is no OnRemoveLiquidity set? you should be able to configure LBP pallet without this